check.go

```diff
@@ -190,6 +198,31 @@ func init() {
 		return nil
 	})
 
+	path = "epp > response > extension > " + ExtFee06 + " chkData"
```

Comment:
Sorry, had I read further, it would have been more obvious that we scan both paths.